Services/UserService/Commands/EditUserCommand.cs

```diff
@@ -29,11 +37,52 @@ public Task<bool> Execute(EditUserRequest request)
                     || string.IsNullOrEmpty(passwordRequest.NewPassword)))
             {
                 passwordChangeValidator.ValidateAndThrow(passwordRequest);
+            } else
```

Comment:
Put `else` in a separate line, not sure if it's in our style convention, but this looks strange

Reply:
Else deleted